source/extensions/filters/network/dubbo_proxy/hessian_deserializer_impl.cc

```diff
@@ -85,6 +78,14 @@ RpcResultPtr HessianDeserializerImpl::deserializeRpcResult(Buffer::Instance& buf
   return result;
 }
 
+void HessianDeserializerImpl::serializeRpcResult(Buffer::Instance& output_buffer,
+                                                 const std::string& content, RpcResponseType type) {
+  uint8_t response_type_code = static_cast<uint8_t>(type);
+  uint8_t base_code = 0x90;
```

Comment:
what meaning? add comments?

Reply:
The encoding for type has been moved to HessianUtils::writeInt.